### DISCUSSION TITLE
Indoor Temp

### DISCUSSION THREAD

**Discussion by thesebastianf** [Blackymas/NSPanel_HA_Blueprint#639] [Ideas]
Hey instead of the internal or another temp for indoor... is there also a possibility to just NOT show an indoor temp?
I would rather see a Min and Max of the outside temp e.g. 

Indoor Temperature Sensor - ENTITY (Optional)
Page "HOME" - An indoor temperature sensor is not necessary. Leave the field empty if you want to use the temperature sensor of the NSPanel. Additionally a temperature correction for the NSPanel sensor is possible under HA Devices. So everyone can adjust the sensor exactly

**Comment by edwardtfn** [comment]
#1724
